prisoner-contact-registry-api-specification.yaml

```diff
@@ -1,109 +1,181 @@
-openapi: "3.0.2"
+openapi: 3.0.1
```

Comment:
It's quite unusual to have the api docs specified in a yaml file. All other projects I have been involved with use annotation.

Reply:
Inherited with the project, I believe has served it purpose and can be removed.